examples/feast/feast_dataflow.py

```diff
@@ -0,0 +1,106 @@
+import json
+import datetime as dt
+import pandas as pd
+
+from kafka import KafkaConsumer, OffsetType
+
+from bytewax import Dataflow, cluster_main, inputs
+
+from feast import FeatureStore
+
+# Configure the feature store for each worker to access
+store = FeatureStore(repo_path=".")
+
+
+def input_builder(worker_index, worker_count):
+    def consume_from_kafka():
+        consumer = KafkaConsumer(bootstrap_servers="localhost:9092", auto_offset_reset=OffsetType.EARLIEST)
```

Comment:
Consumes data from the beginning of the stream. We'll change this if we want to make things streamier

Reply:
Oh yes, I fixed that locally just before the demo. Well, changed it back to a string.